_data/master/navbars/getting-started.yml

```diff
@@ -33,6 +33,8 @@ toc:
       path: /getting-started/kubernetes/installation/vagrant/
   - title: Tutorials
     section:
+    - title: Using calicoctl
```

Comment:
> and "Usage" is for orchestrator independent calico docs.

I'm not entirely convinced that is the case, both in theory and in practice.

See http://docs.projectcalico.org/v2.1/usage/openstack/configuration


That said, I think this is way better than we had it before.  Are you going to update the main Kubernetes index? http://docs.projectcalico.org/master/getting-started/kubernetes/